src/PostgREST/AppState.hs

```diff
@@ -87,6 +92,14 @@ data AppState = AppState
   , stateRetryNextIn              :: IORef Int
   -- | Logs a pool error with a debounce
   , debounceLogAcquisitionTimeout :: IO ()
+  -- | JWT Cache
+  , jwtCache                      :: C.Cache ByteString AuthResult
+  }
```

Comment:
@steve-chavez Here, I have made `jwtCache` a part of `AppState`. Is `AppState` persisted across two subsequent requests from the `spec-tests`? 

Currently I have added a new module `JwtCaching.hs` in the spec-tests where I am making two requests with the same `JWT` token. The `JWT` token is parsed in both requests and the cache doesn't respond with the inserted token in the first request. Hence, I was thinking maybe `AppState` is reset and loses it's contents (jwtCache).